### ISSUE TITLE
Occasionally when making a new direct message, encryption isn't enabled

### ISSUE THREAD

**Issue by artiblaster** [vector-im/element-web#15426]
### Description

Occasionally when making a new direct message, encryption isn't enabled. My understanding is that DMs are supposed to be encrypted by default. But it doesn't ask me if I don't want it encrypted, it just does randomly fails to encrypt the room.

### Steps to reproduce

- make a direct message

Logs being sent: no, can I have the logs sent without my username?

### Version information

For the desktop app:

- **OS**: macOS 10.13.6
- **Version**: Element Version 1.7.8 (1.7.8)

**Comment by t3chguy** [comment]
It uses encryption if the other party has e2ee pubkeys uploaded

**Comment by artiblaster** [comment]
> It uses encryption if the other party has e2ee pubkeys uploaded; if the server doesn't respond due to load it will assume no

Is there a way i can have it try to reach the server again automatically?

**Comment by t3chguy** [comment]
No, it purposefully gives up after one attempt so that it doesn't prevent you from creating the chat if the server is being slow.

**Comment by artiblaster** [comment]
It could do with a better warning system, because this has happened, and I only noticed weeks later.
